jhipster-dependencies/pom.xml

```diff
@@ -69,8 +65,7 @@
         <logstash-logback-encoder.version>6.3</logstash-logback-encoder.version>
         <lz4-java.version>1.7.1</lz4-java.version>
         <mapstruct.version>1.3.1.Final</mapstruct.version>
-        <micrometer.version>1.4.2</micrometer.version>
-        <mongobee.version>0.13</mongobee.version>
+        <mongock.version>4.0.2.alpha</mongock.version>
```

Comment:
@murdos version 4.1.14 is available

Reply:
@DanielFran: we'll handle that after the merge. I really want to fix conflicts and merge the branch as soon as possible.

Reply:
Agree, let's move on